trachoma/trachoma_functions.py

```diff
@@ -378,6 +378,31 @@ def sim_Ind_MDA(params, Tx_mat, vals, timesim, demog, bet, MDA_times, seed, stat
 
     return vals
 
+
+
+
+
+def returnSurveyPrev(vals, TestSensitivity, TestSpecificity):
```

Comment:
Where is/will be this function used?

Reply:
This will be used joint with the next commit, where I have added another function to do the same as sim_Ind_MDA, but with a prevalence survey added onto it to assess how many MDAs should be done, or if we should stop MDA